cypress/e2e/example4-model-esm.cy.ts

```diff
@@ -21,6 +22,15 @@ describe('Example 4 - Model (ESM)', () => {
       .each(($child, index) => expect($child.text()).to.eq(titles[index]));
   });
 
+  it('should expect first row to include "Task 0" and other specific properties', () => {
+    cy.get(`[style="top:${GRID_ROW_HEIGHT * 0}px"] > .slick-cell:nth(1)`).should('contain', 'Task 0');
+    cy.get(`[style="top:${GRID_ROW_HEIGHT * 0}px"] > .slick-cell:nth(2)`).should('contain', '5 days');
```

Comment:
I have done a few changes in these files its simply adding a space between top: and the value and added ; at the end since this is what the default handling comes with

Reply:
yeah I saw that, the test will probably fail after merging your code into my branch, so I can review them afterward. Thanks for the info